hedera-node/src/main/java/com/hedera/services/context/properties/BootstrapProperties.java

```diff
@@ -218,6 +218,11 @@ public Object getProperty(String name) {
 			"tokens.maxPerAccount",
 			"tokens.maxSymbolUtf8Bytes",
 			"tokens.maxTokenNameUtf8Bytes",
+			"nfts.maxMetadataBytes",
+			"nfts.maxBatchSizeBurn",
+			"nfts.maxBatchSizeWipe",
+			"nfts.maxBatchSizeMint",
+			"nfts.maxQueryRange",
```

Comment:
since we are considering nfts as a `subset` of tokens, I think that it would be better if we change  it to `tokens.nfts....`